src/main/java/com/kainos/ea/resources/Capability.java

```diff
@@ -60,4 +69,13 @@ public String getLeadPhoto() {
     public void setLeadPhoto(String leadPhoto) {
         this.leadPhoto = leadPhoto;
     }
+
+    public String getCapabilityJobFamilyName() {
+        return capabilityJobFamilyName;
+    }
+
+    public void setCapabilityJobFamilyName(String capabilityJobFamilyName) {
+        this.capabilityJobFamilyName = capabilityJobFamilyName;
+    }
+
```

Comment:
Move this white space line up to between line 19 & 20 please.

Reply:
Done! Glad You've found it- it could have destroyed the master branch

Reply:
Most certainly. The whole system would be at risk if I hadn't have caught this 😄